File: source/simulation/Station.ts
import { EmergencyVehicle } from "./vehicles/EmergencyVehicle";

export class Station {
  public name: string;
  public vehicles: EmergencyVehicle[];

  constructor(name: string, vehicles?: EmergencyVehicle[]) {
    this.name = name;
    this.vehicles = vehicles || [];
  }
}

File: source/LeitstellenBot.Core.Buildings/Station.cs
using System.Linq;
using System.Collections.Generic;

using LeitstellenBot.Core.Vehicles;

namespace LeitstellenBot.Core.Buildings
{
	public class Station : Building
	{
		public readonly string Name;

		public List<EmergencyVehicle> Vehicles;

		public Station(string name, ICollection<EmergencyVehicle> vehicles)
		{
			Name = name;
			Vehicles = vehicles.ToList();
		}
	}
}